opal/mca/pmix/pmix2x/pmix/src/mca/gds/ds12/gds_dstore.c

```diff
@@ -2804,6 +2808,16 @@ static pmix_status_t dstore_assign_module(pmix_info_t *info, size_t ninfo,
         }
     }
 
+<<<<<<< c632784ca34c467055eadcb4efe84a25e5a3911b:opal/mca/pmix/pmix2x/pmix/src/dstore/pmix_esh.c
```

Comment:
https://github.com/rhc54/ompi/blob/f7e8780a42eb512ed5c54715471168814ef63383/opal/mca/pmix/pmix2x/pmix/src/mca/gds/ds12/gds_dstore.c#L2811:L2829
I'm 100% that this will not compile!